src/components/Identifier/IdentifierLogo/IdentifierLogo.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import classnames from 'classnames'
+
+export interface IdentifierLogoProps {
+  href: string
+  children: React.ReactNode
+  anchorProps?: JSX.IntrinsicElements['a']
+  className?: string
+  anchorClassName?: string
+}
+
+export const IdentifierLogo = ({
+  href,
+  children,
+  anchorProps,
+  className,
+  anchorClassName,
+}: IdentifierLogoProps): React.ReactElement => {
+  const anchorClasses = classnames(
+    'usa-identifier__logo',
+    className,
+    anchorClassName
+  )
```

Comment:
Since the anchor (`<a>`) is the intrinsic element for this component, can we structure this like we do with the other components where additional props (and `className`) are passed along directly to the `<a>` element? I don't think `anchorClassName` is needed here because `className` serves the same purpose, and `anchorProps` can be the used to define the "rest" of the props (ie, `...anchorProps`) instead of a single named prop. For example, to the user it's the difference between:
```
<IdentifierLogo anchorProps={{ ariaLabel: "My logo" }}> ...
```
and:
```
<IdentifierLogo aria-label="My logo"> ...
```